app/assets/javascripts/embargo_reason_field.js

```diff
@@ -0,0 +1,11 @@
+$( document ).ready(function() {
+  $('#embargo_reason_select').change(function() {
+    var val = $("#embargo_reason_select option:selected").val();
+    if (val == 'Other...') {
+      $('#article_embargo_reason').removeClass('hidden');
+    } else {
+      $('#article_embargo_reason').addClass('hidden');
+      $('#article_embargo_reason').val(val);
+    }
+  });
```

Comment:
I feel like when the embargo radio is selected, you would want to trigger "change" initially for the embargo_reason_select